node/node.go

```diff
@@ -1704,6 +1704,10 @@ func (app *App) startAPIServices(ctx context.Context) error {
 			app.Config.API.JSONListener,
 			logger.Zap().Named("JSON"),
 		)
+		// apply cors config
+		// TODO: consider adding ApplyConfig or similar when more config options expected
+		app.jsonAPIServer.CorsAllowedOrigins = app.Config.API.JSONCorsAllowedOrigins
```

Comment:
Pass configuration as parameter to `grpcserver.NewJSONHTTPServer`. If you plan to add more configuration in the future it would make sense to introduce a dedicated `JSONServer` config object as part of `app.Config.API` and pass that object.

Keep in mind that if you change or move existing config values thats a breaking change that is (possibly) incompatible with existing configs, so I would recommend to not move `JSONListener` into that new config object.

Reply:
Ok, Wanted to keep it simple but let it be :)